packages/web/pages/transactions.tsx

```diff
@@ -105,8 +105,9 @@ const Transactions: React.FC = observer(() => {
     ctas: [],
   });
 
-  const [selectedTransaction, setSelectedTransaction] =
-    useState<FormattedTransaction | null>(null);
+  const [selectedTransaction, setSelectedTransaction] = useState<
+    FormattedTransaction | undefined
```

Comment:
IDK if you've seen this comment in other PRs, but IMO things would be simplified and more flexible if we stored IDs (in this case tx hashes) instead of the objects themselves

Reply:
We could use `find(hash)` to return the selected tx, and only render the popover if that is true, which is safer as we aren't at risk for memory leaks, etc.